app/models/item_change_set.rb

```diff
@@ -35,4 +36,11 @@ def initialize
   def changed?
     @changes.any?
   end
+
+  def ==(other)
+    PROPERTIES.each do |property|
+      return false if public_send(property) != other.public_send(property)
+    end
+    true
+  end
```

Comment:
Cool! Can you say more about how this is used?

Reply:
It is used to compare equality of changesets in specs, e.g., https://github.com/sul-dlss/argo/blob/c1f49ec56bc236a8fe3fd4a7963e021c3ea4580c/spec/jobs/set_catkeys_and_barcodes_job_spec.rb#L94-L96

Reply:
Aha, thanks